packages/strapi-design-system/src/Button/utils.js

```diff
@@ -0,0 +1,124 @@
+import { Text } from '../Text';
+import { LIGHT_VARIANTS } from './constants';
+
+export const getVariantColorName = (variant) => {
```

Comment:
Could you please write a test for the utils? Looks like there's a bunch of logic and it may save guts in the future :blush: (or writing snapshots for each scenario from the component entry point?)

Reply:
Sure!